api/lib/domain/usecases/send-sco-invitation.js

```diff
@@ -0,0 +1,49 @@
+const _ = require('lodash');
+const { OrganizationNotFoundError, OrganizationWithoutEmailError, ManyOrganizationsFoundError } = require('../errors');
+const organizationInvitationService = require('../../domain/services/organization-invitation-service');
+
+let errorMessage = null;
+let organizationsFound = null;
+
+module.exports = async function sendSCOInvitation({ UAI, firstName, lastName, locale, organizationRepository, organizationInvitationRepository }) {
+
+  organizationsFound = await organizationRepository.findByExternalId(UAI);
+
+  const nbOrganizations = _.get(organizationsFound, 'length', 0);
+
+  _checkIfManyOrganizationsFoundError(nbOrganizations);
+
+  _checkIfOrganizationNotFoundError(nbOrganizations, organizationsFound);
+
+  _checkIfOrganizationWithoutEmailError(nbOrganizations);
+
+  const email = organizationsFound[0].email;
+  const organizationId = organizationsFound[0].id;
+
+  const organizationSCOInvitation = await organizationInvitationService.createOrganizationSCOInvitation({
+    organizationRepository, organizationInvitationRepository, organizationId, firstName, lastName, email, locale
+  });
+
+  return organizationSCOInvitation;
+};
+
+function _checkIfOrganizationNotFoundError(nbOrganizations) {
+  if ((nbOrganizations == 0) || (nbOrganizations == 1 && 'SCO' !== organizationsFound[0].type)) {
```

Comment:
Organization.js (et c'est là qu'on se rend compte que ce sont des codes techniques...français)
```
const type = {
  SCOLAIRE : 'SCO',
  SUPERIEUR: 'SUP,
  PROFESSIONEL: 'PRO',
};
```

Et ici 
```suggestion
  if ((nbOrganizations == 0) || (nbOrganizations == 1 && Organization.type.SCOLAIRE !== organizationsFound[0].type)) {
```

Reply:
Thanks, j'ai modifié en mettant Organization.types qui expose l'énumération des types avec professionnel avec deux N.